### DISCUSSION TITLE
Save session ?

### DISCUSSION THREAD

**Discussion by cassepipe** [wezterm/wezterm#1665] [Q&A]
I could not find much info on how to manage a workspace/domain (a series of tabs and panes).

How do I save the current one and have it restored next startup on through a command ?

**Comment by wez** [comment]
That isn't possible today.

Here are some issues that track the general feature space:
https://github.com/wez/wezterm/issues/256
https://github.com/wez/wezterm/issues/674

The workspace feature added as part of https://github.com/wez/wezterm/issues/1531 gives you the ability to manage workspaces on a running mux server, so, as long as you don't stop the mux server, your workspace will remain how you left it and you'll be able to reconnect and continue where you left off.

**Reply by neonvoidx** [reply]
any update on this? I was looking at https://github.com/tmux-plugins/tmux-resurrect and would love something like this for wezterms multiplexing

**Reply by OscaRoa** [reply]
To anyone looking for alternatives right now, there's [https://github.com/MLFlexer/resurrect.wezterm](https://github.com/MLFlexer/resurrect.wezterm)
